solutions/devsprint-michel-ferreira-1/src/main/kotlin/com/devpass/challengehexagonal/domain/entity/ClientEntity.kt

```diff
@@ -0,0 +1,19 @@
+package com.devpass.challengehexagonal.domain.entity
+
+import java.time.LocalDateTime
+import javax.persistence.*
+
+@Entity
+data class ClientEntity(
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    val id: Long? = null,
+
+    val name: String,
+
+    val birthDate: LocalDateTime,
+
+    val account: AccountEntity,
+
+//    @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true)
+//    @JoinColumn(name = "account_id", referencedColumnName = "account")

```

Comment:
Tivemos dúvidas para usar as anotações @OneToMany/@ManyToOne nessa parte, pois sempre aparece um erro (This annotation is not applicable to target 'value parameter'), vc poderia dar um help, @michel05 ? 😄